firebase-functions/firebase-functions.gradle

```diff
@@ -63,6 +69,7 @@ dependencies {
     implementation 'com.google.firebase:firebase-iid-interop:17.0.0'
 
     implementation 'com.squareup.okhttp3:okhttp:3.12.1'
+    implementation 'commons-validator:commons-validator:1.7'
```

Comment:
I think we can get rid of this new dependency (see comments below)

Reply:
Done